src/components/data/DataTable.js

```diff
@@ -222,13 +226,33 @@ const DataTable = props => {
     return entities.length && _.every(k => _.includes(k, selectedKeys), entityKeys)
   }
 
+  const searchByColumn = (field, v) => {
+    setActiveTextFilter('')
+    setColumnFilter(`${field}=${v.toString().trim()}`)
```

Comment:
I don't feel very strongly about this, so feel free to ignore: you could store `columnFilter` into state as an object e.g. `{ filterColAttr, filterColTerm }`; this prevents potentially-fragile string parsing in `parseColumnFilter`. Keep it as a structured object until creating the `queryOptions` object to pass to ajax; at that point, turn it into a string.